### PR TITLE
zh-trans: update schedule GPUs

### PR DESCRIPTION
**Pull request by mysunshine92**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.15 Features: set Milestone to 1.15 and Base Branch to dev-1.15
> 
> For pull requests on Chinese localization, set Base Branch to release-1.14
>
> For pull requests on Korean Localization: set Base Branch to dev-1.14-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by mysunshine92** [comment]
cc @markthink

**Comment by mysunshine92** [comment]
cc @tengqm

**Comment by chenrui333** [comment]
will review later when I have time. 😄 👍

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 1]
```diff
@@ -1,44 +1,40 @@
...
-
--->
-
-
+--->
 Kubernetes 支持对节点上的 AMD 和 NVIDA GPU 进行管理，目前处于**实验**状态。对 NVIDIA GPU 的支持在 v1.6 中加入，已经经历了多次不向后兼容的迭代。而对 AMD GPU 的支持则在 v1.9 中通过 [device plugin](#deploying-amd-gpu-device-plugin) 加入。
 
-这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制。
+这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制
+。
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion

```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 2]
```diff
@@ -1,44 +1,40 @@
...
 and the current limitations.
-
--->
-
-
+--->
 Kubernetes 支持对节点上的 AMD 和 NVIDA GPU 进行管理，目前处于**实验**状态。对 NVIDIA GPU 的支持在 v1.6 中加入，已经经历了多次不向后兼容的迭代。而对 AMD GPU 的支持则在 v1.9 中通过 [device plugin](#deploying-amd-gpu-device-plugin) 加入。
 
-这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制。
+这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
这个页面介绍了用户如何在不同的 Kubernetes 版本中使用 GPU，以及当前存在的一些限制。
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 3]
```diff
@@ -117,37 +112,32 @@ spec:
...
-### 部署 AMD GPU device plugin
-
-[官方的 AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin) 有以下要求：
-
+--->
 - Kubernetes 节点必须预先安装 AMD GPU 的 Linux 驱动。
 
 如果你的集群已经启动并且上述要求满足的话，可以这样部署 AMD device plugin：
-
+requirements are satisfied:
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]: 这个没有翻译？

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 4]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
+  * CZ - Carrizo
+  * AI - Arctic Islands
+  * RV - Raven
+
+Example result:
+--->
+* 设备 ID (-device-id)
+* VRAM 大小 (-vram)
+* SIMD 数量(-simd-count)
+* 计算单位(-cu-count)
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
* 计算单位数量 (-cu-count)
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 5]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
+
+If different nodes in your cluster have different types of GPUs, then you
 can use [Node Labels and Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/)
 to schedule pods to appropriate nodes.
 
 For example:
+--->
+## 集群内存在不同类型的 NVIDIA GPU
 
+如果集群内部的不同节点上有不同类型的 NVIDIA GPU，那么你可以使用 [Node Labels 和 Node Selectors](/docs/tasks/configure-pod-container/assign-pods-nodes/) 来将pod调度到合适的节点上。
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
如果集群内部的不同节点上有不同类型的 NVIDIA GPU，那么你可以使用 [节点标签和节点选择器](/docs/tasks/configure-pod-container/assign-pods-nodes/) 来将 pod 调度到合适的节点上。
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 6]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
+
+举一个例子：
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
例如：
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 7]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
 ```shell
 # 为你的节点加上它们所拥有的加速器类型的标签
 kubectl label nodes <node-with-k80> accelerator=nvidia-tesla-k80
 kubectl label nodes <node-with-p100> accelerator=nvidia-tesla-p100
 ```
 
+<!--
+For AMD GPUs, you can deploy [Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller), which automatically labels your nodes with GPU properties. Currently supported properties:
+--->
+对于 AMD GPUs，您可以部署 [Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller)，它会自动给节点打上 GPU 属性标签。目前支持的属性：
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
对于 AMD GPUs，您可以部署 [节点标签器](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller)，它会自动给节点打上 GPU 属性标签。目前支持的属性：
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 8]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
 ```
 # 在 Container-Optimized OS 上安装 NVIDIA 驱动:
-kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/k8s-1.9/daemonset.yaml
+kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/stable/daemonset.yaml
 
 # 在 Ubuntu 上安装 NVIDIA 驱动 (实验性质):
-kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/k8s-1.9/nvidia-driver-installer/ubuntu/daemonset.yaml
+kubectl create -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/stable/nvidia-driver-installer/ubuntu/daemonset.yaml
 
 # 安装 device plugin:
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
# 安装设备插件:
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 9]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
-
-[GKE/GCE 使用的 NVIDIA GPU device plugin](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
-并不要求使用 nvidia-docker，并且对于任何实现了 Kubernetes CRI 的容器运行时，都应该能够使用。这一实现已经在 [Container-Optimized OS](https://cloud.google.com/container-optimized-os/) 上进行了测试，并且在 1.9 版本之后会有对于 Ubuntu 的实验性代码。
-
-在你 1.9 版本的集群上，你能使用下面的命令来安装 NVIDIA 驱动以及 device plugin：
+--->
+在你 1.12 版本的集群上，你能使用下面的命令来安装 NVIDIA 驱动以及 device plugin：
 
 ```
 # 在 Container-Optimized OS 上安装 NVIDIA 驱动:
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
# 在容器优化的操作系统上安装 NVIDIA 驱动:
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 10]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
-#### NVIDIA GPU device plugin used by GKE/GCE
-
-The [NVIDIA GPU device plugin used by GKE/GCE](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
+The [NVIDIA GPU device plugin used by GCE](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
 doesn't require using nvidia-docker and should work with any container runtime
 that is compatible with the Kubernetes Container Runtime Interface (CRI). It's tested
 on [Container-Optimized OS](https://cloud.google.com/container-optimized-os/)
 and has experimental code for Ubuntu from 1.9 onwards.
+--->
+#### GCE 中使用的 NVIDIA GPU device plugin
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
#### GCE 中使用的 NVIDIA GPU 设备插件
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 11]
```diff
@@ -216,84 +208,137 @@ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v1.
...
 
-On your 1.9 cluster, you can use the following commands to install the NVIDIA drivers and device plugin:
+[GCE 使用的 NVIDIA GPU device plugin](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu) 并不要求使用 nvidia-docker，并且对于任何实现了 Kubernetes CRI 的容器运行时，都应该能够使用。这一实现已经在 [Container-Optimized OS](https://cloud.google.com/container-optimized-os/) 上进行了测试，并且在 1.9 版本之后会有对于 Ubuntu 的实验性代码。
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
[GCE 使用的 NVIDIA GPU 设备插件](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu) 并不要求使用 nvidia-docker，并且对于任何实现了 Kubernetes CRI 的容器运行时，都应该能够使用。这一实现已经在 [容器优化的操作系统](https://cloud.google.com/container-optimized-os/) 上进行了测试，并且在 1.9 版本之后会有对于 Ubuntu 的实验性代码。
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 12]
```diff
@@ -177,7 +178,15 @@ has the following requirements:
 
 To deploy the NVIDIA device plugin once your cluster is running and the above
 requirements are satisfied:
+--->
+- Kubernetes 的节点必须预先安装了 NVIDIA 驱动
+- Kubernetes 的节点必须预先安装 [nvidia-docker 2.0](https://github.com/NVIDIA/nvidia-docker)
+- Docker 的[默认运行时](https://github.com/NVIDIA/k8s-device-plugin#preparing-your-gpu-nodes)必须设置为 nvidia-container-runtime，而不是 runc
+- NVIDIA 驱动版本 ~= 361.93
+
+如果你的集群已经启动并且满足上述要求的话，可以这样部署 NVIDIA device plugin：
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
如果你的集群已经启动并且满足上述要求的话，可以这样部署 NVIDIA 设备插件：
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 13]
```diff
@@ -168,7 +160,16 @@ There are currently two device plugin implementations for NVIDIA GPUs:
 
 The [official NVIDIA GPU device plugin](https://github.com/NVIDIA/k8s-device-plugin)
 has the following requirements:
+--->
+### 部署 NVIDIA GPU device plugin
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
### 部署 NVIDIA GPU 设备插件
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 14]
```diff
@@ -168,7 +160,16 @@ There are currently two device plugin implementations for NVIDIA GPUs:
...
+
+对于 NVIDIA GPUs，目前存在两种 device plugin 的实现：
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
对于 NVIDIA GPUs，目前存在两种设备插件的实现：
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 15]
```diff
@@ -168,7 +160,16 @@ There are currently two device plugin implementations for NVIDIA GPUs:
...
 
+#### 官方的 NVIDIA GPU device plugin
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
#### 官方的 NVIDIA GPU 设备插件
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 16]
```diff
@@ -168,7 +160,16 @@ There are currently two device plugin implementations for NVIDIA GPUs:
...
+
+[官方的 NVIDIA GPU device plugin](https://github.com/NVIDIA/k8s-device-plugin) 有以下要求:
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
[官方的 NVIDIA GPU 设备插件](https://github.com/NVIDIA/k8s-device-plugin) 有以下要求:
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 17]
```diff
@@ -156,10 +146,12 @@ kubectl create -f https://raw.githubusercontent.com/RadeonOpenCompute/k8s-device
 kubectl create -f https://raw.githubusercontent.com/RadeonOpenCompute/k8s-device-plugin/r1.10/k8s-ds-amdgpu-dp.yaml
 ```
 
+<!--
+Report issues with this device plugin to [RadeonOpenCompute/k8s-device-plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin).
+--->
 请到 [RadeonOpenCompute/k8s-device-plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin) 报告有关此 device plugin 的问题。
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
请到 [RadeonOpenCompute/k8s-device-plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin) 报告有关此设备插件的问题。
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 18]
```diff
@@ -117,37 +112,32 @@ spec:
 ```
 
 <!--
-
 ### Deploying AMD GPU device plugin
 
 The [official AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 has the following requirements:
+--->
+### 部署 AMD GPU device plugin
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
### 部署 AMD GPU 设备插件
```

File: content/zh/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 19]
```diff
@@ -48,31 +44,36 @@ To enable GPU support through device plugins before 1.10, the `DevicePlugins`
...
 
 <!--
+Then you have to install GPU drivers from the corresponding vendor on the nodes
+and run the corresponding device plugin from the GPU vendor
+([AMD](#deploying-amd-gpu-device-plugin), [NVIDIA](#deploying-nvidia-gpu-device-plugin)).
 
 When the above conditions are true, Kubernetes will expose `nvidia.com/gpu` or
 `amd.com/gpu` as a schedulable resource.
+--->
+接着你需要在主机节点上安装对应厂商的 GPU 驱动并运行对应厂商的 device plugin ([AMD](#deploying-amd-gpu-device-plugin)、[NVIDIA](#deploying-nvidia-gpu-device-plugin))。
```
Comment (chenrui333) [inline content/zh/docs/tasks/manage-gpus/scheduling-gpus.md]:
```suggestion
接着你需要在主机节点上安装对应厂商的 GPU 驱动并运行对应厂商的设备插件 ([AMD](#deploying-amd-gpu-device-plugin)、[NVIDIA](#deploying-nvidia-gpu-device-plugin))。
```

**Comment by mysunshine92** [comment]
> 
> 
> lgtm except `device plugin` -> `设备插件`

谢谢你的评审，我已经全部修改了，麻烦给打下lgtm 和 approve 标签。

**Comment by chenrui333** [comment]
@mysunshine92 Thanks for your contributions, 🎉 🎉 🎉

**Pull request merged**